### PR TITLE
[KD] Add Footer

### LINKED ISSUE #49: Add Footer
## Summary
Add a footer to the bottom of the website that includes attributions to the application's creators.

## Acceptance criteria
- [x] Make a footer that is visible on every page of the site
- [x] Link to each creator's GitHub profile

## Notes
Sub-issue of https://github.com/the-collab-lab/tcl-69-smart-shopping-list/issues/14

### PR DESCRIPTION
**Pull request by piecanoe**
## Description

This code adds a footer to the bottom of the site that is visible from every page. It also includes an attribution that links to each contributor's GitHub account.

Note: The Navbar has been moved to the top of the page to clear space for the footer.

## Related Issue
Closes #49
Sub-issue of #14

## Acceptance Criteria
- [x] Make a footer that is visible on every page of the site
- [x] Link to each creator's GitHub profile

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.

### Before

before_1

### After

after_1

Footer is not sticky:

https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/122697843/ce984a0d-d7da-49eb-8f28-e3087301463e

Footer text stays center-aligned even on screens with small width:
after_3

## Testing Steps / QA Criteria

1.  On Home, make sure you can see the footer at the bottom of the page
2.  Navigate to the List page and shrink your window size so you must scroll to see the full page. Scroll down to the bottom of the page to see the footer.
3. Shrink the window width to see that the attribution text stays center-aligned even as it wraps to the next line.

### CODE REVIEW

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -37,6 +37,60 @@ export function Layout() {
 						</NavLink>
 					</div>
 				</nav>
+				<footer className="Layout-footer">
+					<p className="Layout-footer-attribution">
+						Crafted with care{' '}
+						<span role="img" aria-label="heart">
+							❤️
+						</span>
+						by{' '}
```
Comment (hsiangj) [inline src/views/Layout.jsx]: Super nit: perhaps a space after the emoji?

**Review by hsiangj — APPROVED** [review]
Great work! Links also work perfectly.

File: src/views/Layout.jsx (hunk 1 above)
Comment (piecanoe) [reply src/views/Layout.jsx]: Good catch. Will update!

**Review by krsnamara — APPROVED** [review]
Great job Karen!

**Pull request merged**